dashboard/app/models/pd/application/application_base.rb

```diff
@@ -103,7 +103,7 @@ class ApplicationBase < ActiveRecord::Base
       ]
     }
 
-    has_many :emails, class_name: 'Pd::Application::Email'
+    has_many :emails, class_name: 'Pd::Application::Email', foreign_key: 'pd_application_id'
```

Comment:
This association generated invalid queries sometimes (not sure why sometimes) because it wasn't explicitly specifying the foreign key which does not follow Rails conventions.  Note that the inverse association did specify this foreign key.  It's possible that when both are loaded, this one infers the right key column from that one?

https://github.com/code-dot-org/code-dot-org/blob/e4ae2256fbac246dfef0adc3e706f9d8cfa33d61/dashboard/app/models/pd/application/email.rb#L22